### PR TITLE
Rancher server as module

### PR DESCRIPTION
**Pull request by jeffb4**

### CODE REVIEW

File: rancher-server/locals.tf [hunk 1]
```diff
@@ -1,8 +1,9 @@
 locals {
-  name              = "rancher-demo"
+  name              = var.name
   rancher_version   = "v2.2.8"
```
Comment (drpebcak) [inline rancher-server/locals.tf]: `rancher_version` should probably also be configurable for this to be a good module. Along with that, the helm repo that is being used should also be configurable

**Review by drpebcak — CHANGES_REQUESTED** [review]
Do you have an example that uses this code as a module? I'd like to see how you envision someone consuming this.

File: rancher-server/locals.tf (hunk 1 above)
Comment (jeffb4) [reply rancher-server/locals.tf]: parameterized chart and version.

**Comment by jeffb4** [comment]
Documentation of module (with an example snippet) added

**Review by drpebcak — APPROVED** [review]

**Pull request merged**